### ISSUE TITLE
Bulged cube test fails sometimes

### ISSUE THREAD

**Issue by kidder** [sxs-collaboration/spectre#510]
# Bug reports:

Bulged cube:  equidistant test fails GCC release build

### Current behavior:

https://travis-ci.org/sxs-collaboration/spectre/jobs/338151890

**Comment by nilsdeppe** [comment]
When adding the following in RootFinder.hpp:

```
try {
      auto result = boost::math::tools::toms748_solve(
          [&f, i](double x) { return f(x, i); },
          lower_bound[i] - absolute_tolerance,
          upper_bound[i] + absolute_tolerance, tol, max_iter);
      result_vector[i] = result.first + 0.5 * (result.second - result.first);
    } catch (...) {
      Parallel::abort("Here");
    }
```
we hit the catch block, so the boost call is likely failing and we are not handling the error correctly.

**Comment by nilsdeppe** [comment]
fixed in #513
